hera_mc/mc.py

```diff
@@ -110,6 +110,20 @@ def get_mc_argument_parser():
                    help='Name of the database to connect to. The default is used if unspecified.')
     return p
 
+def get_cm_csv_path(args):
```

Comment:
I'll finish this shortly.  It does not currently exist.  I plan to do this tomorrow.